File: output/llama3.1-8b/avatar/Python/Java/codeforces_387_B.java
import java.util.Scanner;

public class codeforces_387_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String[] input = scanner.nextLine().split(" ");
        int n = Integer.parseInt(input[0]);
        int m = Integer.parseInt(input[1]);

        String[] roundComplexityInput = scanner.nextLine().split(" ");
        int[] roundComplexity = new int[n];
        for (int i = 0; i < n; i++) {
            roundComplexity[i] = Integer.parseInt(roundComplexityInput[i]);
        }

        String[] georgeComplexityInput = scanner.nextLine().split(" ");
        int[] georgeComplexity = new int[m];
        for (int i = 0; i < m; i++) {
            georgeComplexity[i] = Integer.parseInt(georgeComplexityInput[i]);
        }

        int i = 0;
        int j = 0;
        while (i < n && j < m) {
            if (roundComplexity[i] <= georgeComplexity[j]) {
                i++;
            }
            j++;
        }
        System.out.println(n - i);
    }
}

File: output/llama3.1-8b_IO_1/avatar/Java/Python/codeforces_387_B.py
n, m = map(int, input().split())
req = list(map(int, input().split()))[:n]
pre = list(map(int, input().split()))[:m]

i, j = n - 1, m - 1
ans = 0
while i >= 0 and j >= 0:
    if req[i] > pre[j]:
        ans += 1
        j -= 1
    else:
        i -= 1

print(ans)
